src/node_report.cc

```diff
@@ -597,7 +721,9 @@ static void WriteNodeReport(Isolate* isolate, DumpEvent event, const char* messa
   out << "\n================================================================================";
   out << "\n==== Node.js libuv Handle Summary ==============================================\n";
   out << "\n(Flags: R=Ref, A=Active)\n";
-  out << "\nFlags  Type      Address\n";
+  out << std::left << std::setw(7) << "Flags" << std::setw(10) << "Type"
+      << std::setw(4 + 2 * sizeof(void*)) << "Address" << "Details"
+      << std::endl;
```

Comment:
I've been caught out by setting formatting flags before, they are set on the stream so any other users will be affected by them later on. (It's a problem if we are writing to stdout or stderr!)

Using them is fine but I think we should probably call:
`std::ios_base::fmtflags format = out.flags();`
to save them at the start of WriteNodeReport and:
`out.flags(format);`
so we leave the streams as we found them. (I think that will reset things like fill characters but we should probably test it to be sure.)